components/Cards/ResourceCard/ResourceCard.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { string, number, bool, func } from 'prop-types';
+import classnames from 'classnames';
+
+import Card from 'components/Cards/Card/Card';
+import ThumbsUp from 'static/images/icons/FontAwesome/thumbs-up.svg';
+import ThumbsDown from 'static/images/icons/FontAwesome/thumbs-down.svg';
+import styles from './ResourceCard.css';
+
+ResourceCard.propTypes = {
+  name: string.isRequired,
+  imageSource: string.isRequired,
```

Comment:
Have we discussed where or how we're going to decide on what images we want for each resource? Perhaps it makes sense to add images to the resources db itself, but we'd have to do some migrations and then do the tedious work of determining which images to stick with which resources.

Do we want to try to base it on language and/or category? If so, we might be able to just pull from some static assets somewhere.

In the meantime, I'm a bit worried about making the images required. Will the entire card break down without an image for a given resource?

Reply:
> Do we want to try to base it on language and/or category?

Seems like a good idea to me.

> In the meantime, I'm a bit worried about making the images required. Will the entire card break down without an image for a given resource?

Maybe use a placeholder as a fallback?  It would probably make it easier to keep the styling consistent.

Reply:
We could use the Operation Code logo as a fallback

Reply:
We'll create an image mapping for every language and dynamically select the logo on the front-end.